api/src/pcapi/sandboxes/scripts/creators/industrial/create_industrial_venues.py

```diff
@@ -52,51 +61,49 @@ def create_industrial_venues(offerers_by_name: dict) -> dict[str, Venue]:
 
     for offerer_index, (offerer_name, offerer) in enumerate(offerers_by_name.items()):
         geoloc_match = re.match(r"(.*)lat\:(.*) lon\:(.*)", offerer_name)
+        latitude = float(geoloc_match.group(2)) if geoloc_match else None
+        longitude = float(geoloc_match.group(3)) if geoloc_match else None
 
         venue_name = MOCK_NAMES[mock_index % len(MOCK_NAMES)]
 
         # create all possible cases:
         # offerer with or without iban / venue with or without iban
+        iban = None
+        bic = None
         if offerer.iban:
             if iban_count == 0:
                 iban = iban_prefix
                 bic = bic_prefix + str(bic_suffix)
                 iban_count = 1
             elif iban_count == 2:
-                iban = None
-                bic = None
                 iban_count = 3
         else:
             if iban_count in (0, 1):
                 iban = iban_prefix
                 bic = bic_prefix + str(bic_suffix)
                 iban_count = 2
             elif iban_count == 3:
-                iban = None
-                bic = None
                 iban_count = 0
 
-        if offerer_index % OFFERERS_WITH_PHYSICAL_VENUE_REMOVE_MODULO:
+        if offerer_index % OFFERERS_WITH_PHYSICAL_VENUE_REMOVE_MODULO == 0:
+            venue = None
+        else:
             if offerer_index % OFFERERS_WITH_PHYSICAL_VENUE_WITH_SIRET_REMOVE_MODULO:
                 comment = None
                 siret = f"{offerer.siren}11111"
             else:
                 comment = "Pas de siret car c'est comme cela."
                 siret = None
 
-            # TODO: remove venue_type and label to code mapping when
-            # the venue_type table has been finally replaced by the
-            # VenueTypeCode enum
-
             venue = offerers_factories.VenueFactory(
                 managingOfferer=offerer,
-                bookingEmail="fake@example.com",
-                latitude=float(geoloc_match.group(2)),  # type: ignore [union-attr]
-                longitude=float(geoloc_match.group(3)),  # type: ignore [union-attr]
+                bookingEmail=f"{slugify(venue_name)}@venue.com",
```

Comment:
[discussion] Souhaite-t-on vraiment ajouter une dépendance à une nouvelle bibliothèque Python, juste pour ces 2 appels dans la sandbox ? Est-ce qu'on ne pourrait pas écrire quelque chose du genre : 

```suggestion
                bookingEmail=f"booking-email@venue{venue.id}.example.com",
```

Reply:
Je trouvais ça plus sympa mais soit, simplifions

Reply:
Sachant qu'on n'a pas l'id du lieu avant de le créer, ce sera donc : 
`bookingEmail=f"booking-email@offerer{offerer.id}.example.com"`
Commit imminent.